src/GameData.ts

```diff
@@ -156,9 +195,62 @@ export default class GameData {
         if (typeof result === "string") {
             message.channel.send(result);
             return;
+        } else {
+            message.channel.send(this.buildEmbed(result));
+        }
+    }
+
+    public buildEmbed(rawData: EmbeddableDatum) {
+        const embed = new Discord.RichEmbed();
+        switch (rawData.type) {
+            case "ChampionData": {
+                const imageString = `https://raw.communitydragon.org/latest/plugins/rcp-be-lol-game-data/global/default/v1/champion-icons/${rawData.id}.png`;
```

Comment:
I'd prefer you use cdn.communitydragon.org here. If the path for these images ever changes, we won't need to update (because stelar will fix dragonrouter for us).

Reply:
https://cdn.communitydragon.org/latest/champion/{key}/square

Reply:
I chose to use raw because as stelar noted, there's no cdn for items or perks. initially before submitting this pr i was looking into helping with that, but from what i understand dragonrouter needs a larger scale refactor.

I also chose raw for consistency, but I can switch to cdn if thats preferable